### ISSUE TITLE
Would be cool to have a show in github link that opens in-app browser of the item you’re on from the middle-top dropdown

### ISSUE THREAD

**Issue by ijm8710** [GitHawkApp/GitHawk#2305]
Currently the only methods are to open directly to safari or to open repo home and the have to navigate to specific ticket again

**Comment by rnystrom** [comment]
Would this be adding the same functionality that’s in the share sheet but for in-app?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ijm8710** [comment]
Exactly. If you feel that’s not necessary  feel free to close (won’t pushback as much on this ticket :) ) but I’m a big in-app fan and it relates to the other two options you have in the middle of the top bar

**Comment by rnystrom** [comment]
I'm impartial, but I'd prefer we only open links one way throughout (in-app or external). If there's someone that wants to own this that's fine.
